msiexec: add page (#7513)
tldr-pages/tldr: pages/windows/msiexec.md

# msiexec

> > Install, update, repair, or uninstall Windows programs through MSI and MSP apckage files.

Reviewer: typo? Also, would this read better?
```suggestion
> Install, update, repair, or uninstall Windows programs using MSI and MSP package files.
```

Author: Yeah

Reviewer: I think you forgot to apply this change, @reinhart1010?

> > More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec>.

Reviewer 2: We don't use the i18n param in Microsoft URLs, it's auto detected based on user's language config
```suggestion
> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/msiexec>.
```

- Install a program from its MSI package:

`msiexec /package {{path/to/file.msi}}`

- Install a MSI package from a website:

`msiexec /package {{https://example.com/installer.msi}}`

- Install a MSP patch file:

`msiexec /update {{path/to/file.msp}}`

> - Uninstall a program or patch through their respective MSI or MSP file:

Reviewer: Perhaps this will read better?
```suggestion
- Uninstall a program or patch using their respective MSI or MSP file:
```
